trajectory_tracker/test/src/test_trajectory_tracker.cpp

```diff
@@ -31,7 +31,90 @@
 #include <string>
 #include <vector>
 
-#include <trajectory_tracker_test.h>
+#include "trajectory_tracker_test.h"
+
+using std::chrono::duration;
+using std::chrono::duration_cast;
+using std::chrono::nanoseconds;
+
+class RosRate : public rclcpp::RateBase
```

Comment:
`rclcpp::Rate` ignores `use_sim_time` option, so I added a new rate class to use the ROS-time clock.